examples/website/src/shared/utility.ts

```diff
@@ -113,3 +113,27 @@ export function createStyleRenderer() {
     plugins: [felaPrefixer(), felaFallbackValue()]
   })
 }
+
+export function transformCssStringToObject(styleCustom: any) {
+  const styleRules = !!styleCustom ? styleCustom.split(';') : []
```

Comment:
@roosnic1 this function is exists 2 times, due i assume the website should work independent from Editor and api

Reply:
Explanation: I tried the approach to store a object first, but changed it back due i see i would be a limitation and type inconvenience to store a string style value as object.

2 main reason:

- if you think beyond React or VueJS implementations, keep the original type
- currently in editor, it would mean to retransform string to object on each input

Reply:
the values are taken from a real iframe html element, ` styleCustom: !!iframe.style && !!iframe.style.cssText ? iframe.style.cssText : ''`

This and the transform function will ensure that only css properties with valid values will be set (e..g you cannot save wron css)